### DISCUSSION TITLE
Save output files without subfolders

### DISCUSSION THREAD

**Discussion by Kazaa3** [audiamus/AaxAudioConverter#87] [Ideas]
Hallo audiamus,

 could you please add an option to save the output files (e.g. single m4b files) in a main folder without any subfolders.

**Comment by audiamus** [comment]
Difficult and unlikely, I'm afraid. It would only be applicable to single file outputs, as you say. So even a two-part book would not fit. That's confusing for a user. The other obstacle is file name uniqueness, not guaranteed due to the available naming options. A similar constraint already exists for one of the folder settings. And even I as the developer find this hard to remember.
